debian/control

```diff
@@ -127,6 +127,8 @@ Architecture: linux-any
 Multi-Arch: same
 Depends: ibverbs-providers (= ${binary:Version}),
          libibverbs1 (= ${binary:Version}),
+         libnl-3-dev,
+         libnl-route-3-dev,
```

Comment:
rdma-core section has already this dependency,
Isn't this enough? sorry for my dumb question, I'm not debian guru.

Source: rdma-core
Maintainer: Linux RDMA Mailing List <linux-rdma@vger.kernel.org>
Uploaders: Benjamin Drung <benjamin.drung@profitbricks.com>,
           Talat Batheesh <talatb@mellanox.com>
Section: net
Priority: optional
Build-Depends: cmake (>= 2.8.11),
               debhelper (>= 9),
               debhelper (>= 9.20160709) | dh-systemd,
               dpkg-dev (>= 1.17),
               libnl-3-dev,
               libnl-route-3-dev,
               libudev-dev,

Reply:
Yes - in that section that's a build dependency for these packages. Here it's a runtime dependency of the dev library, and as the commit msg says it's needed if a third party wants to do static linking with ibverbs, as they'll need to install and static link to everything that ibverbs depends on, which in this case is nl3

Reply:
No, this is wrong, we don't support static linking in Debian packaging today so don't need the -dev packages here. If we add static libraries someday it would become necessary.

Reply:
The static library is built and shipped as you can see here:
https://packages.debian.org/buster/amd64/libibverbs-dev/filelist

Reply:
packages.debian.org is not working, it is showing you a file list from a pre rdma-core package, our packaging does not enable static libraries. download the .deb from buster and check yourself with dpkg-deb -c.

Reply:
Uh, hadn't noticed it was broken, interesting (and annoying). I'll revert this commit then, thanks for spotting it.